src/main/java/com/blackhearth/blockchain/wallet/BasicWalletService.java

```diff
@@ -0,0 +1,12 @@
+package com.blackhearth.blockchain.wallet;
+
+import com.blackhearth.blockchain.wallet.signature.SignatureUtils;
+
+public class BasicWalletService implements WalletService {
+    @Override
+    public String setWalletHash(Wallet wallet) {
```

Comment:
get*

ta funkcja nic nie ustawia. lepsze słowo to calculate, a nazwa klasy to WalletHashCalculator, czy coś.

Reply:
w sensie get albo calculate przy czym calculate jest lepsze.